### PR TITLE
Adjust redis connector with latest syntax changes

### PR DESCRIPTION
**Pull request by Manuri**

### CODE REVIEW

File: component/src/main/ballerina/redis/redis_client.bal [hunk 1]
```diff
@@ -0,0 +1,707 @@
+package ballerina.redis;
```
Comment (anupama-pathirage) [inline component/src/main/ballerina/redis/redis_client.bal]: Shall we add license header?

File: component/src/main/ballerina/redis/redis_client_endpoint.bal [hunk 2]
```diff
@@ -0,0 +1,69 @@
+package ballerina.redis;
```
Comment (anupama-pathirage) [inline component/src/main/ballerina/redis/redis_client_endpoint.bal]: Shall we add license header?

Comment (Manuri) [reply component/src/main/ballerina/redis/redis_client_endpoint.bal]: Added in https://github.com/wso2-ballerina/package-redis/pull/9/commits/8bfc4913a7e31f674f955add1570b981063422ec

File: component/src/main/ballerina/redis/redis_client.bal (hunk 1 above)
Comment (Manuri) [reply component/src/main/ballerina/redis/redis_client.bal]: Added in https://github.com/wso2-ballerina/package-redis/pull/9/commits/8bfc4913a7e31f674f955add1570b981063422ec

**Pull request merged**